content/blog/analyzing-the-success-of-the-hpe-dev-workshops-on-demand.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Analyzing the success of the HPE DEV Workshops-on-Demand
+date: 2021-09-16T07:14:18.418Z
+author: Frederic Passeron
+authorimage: /img/fp-hpewod.jpg
+thumbnailimage: /img/fp-hpewod.jpg
+---
+![](/img/wodanalysisblog1.png "HPE DEV Hackshack")
+
+In October of 2020, the HPE DEV team introduced a **very easy** way for users to learn about a variety of different software topics by interacting with code virtually – in the form of **free** HPE DEV Workshops-on-Demand. Starting with just three workshops, the program has continued to grow and now offers over 20 courses, ranging from basic coding 101 to sophisticated infrastructure automation.
+
+![](/img/wodanalysisblog2.png)
+
+These hands-on workshops cover a variety of [topics](https://hackshack.hpedev.io/workshops), including open source technologies (i.e. Kubernetes, SPIFFE/SPIRE, Grommet), popular programming languages (i.e. Python, RUST, StackStorm), coding tools (i.e. APIs, Git), and market-leading HPE technologies (i.e. HPE Ezmeral Container Platform and Data Fabric). Whether it’s due to the popularity of these topics, the workshop’s unique, easy-to-follow methodology, the associated badge recognition program, or the fact that they’re free – these workshops have certainly gained quite a lot of interest!
```

Comment:
I would add HPE OneView and HPE iLO/Redfish to the HPE Technologies. On it's own Ezmearal doesnt stand as a "industry leading" technology. We are developers, we are humble and authentic.